cmd/parca-agent/main.go

```diff
@@ -60,6 +61,7 @@ var (
 	commit  string
 	date    string
 	builtBy string
```

Comment:
nit: I saw that it's not used anymore, let's remove `builtBy`, WDTY?

Reply:
Oh, thought it was used by `goreleaser`! Happy to remove it otherwise!